docs/static_site/src/pages/api/faq/env_var.md

```diff
@@ -375,7 +375,7 @@ If ctypes is used, it must be `mxnet._ctypes.ndarray.NDArrayBase`.
   - This variable controls how many CuDNN dropout state resources to create for each GPU context for use in operator.
 
 * MXNET_SUBGRAPH_BACKEND
-  - Values: String ```(default="MKLDNN")``` if ONEDNN is avaliable, otherwise ```(default="")```
+  - Values: String ```(default="DNNL")``` if ONEDNN is avaliable, otherwise ```(default="")```
```

Comment:
```suggestion
  - Values: String ```(default="DNNL")``` if oneDNN is available, otherwise ```(default="")```
```

Reply:
I would suggest to use `ONEDNN` for the backend name. There's no need to introduce naming with `DNNL` unless it was already there.